core/templates/pages/landing-pages/stewards-landing-page/stewards-landing-page.component.html

```diff
@@ -123,10 +123,10 @@ <h2 class="oppia-stewards-landing-h2" style="padding-right: 15px;">Support from
     </div>
   </div>
 
-  <div class="oppia-stewards-landing-section" style="background-color: #e8e7e3; max-width: 90%; padding-bottom: 50px;">
-    <section style="padding: 30px;">
+  <div class="oppia-stewards-landing-section oppia-stewards-landing-section-div">
```

Comment:
You really need more specific names here. In particular, names should specifically identify the item they are attached to. If you call this a "div" -- then wouldn't other divs in this page qualify too? That is a sign that the name you have chosen here is too vague.